src/infra/ingest-datafusion/tests/tests/test_reader_shapefile.rs

```diff
@@ -251,7 +251,7 @@ async fn test_read_shapefile_geom() {
                 serde_json::json!({
                     "type": "MultiPolygon",
                     "coordinates": [[[
-                        [30.466304779052763, 50.58700942993181], [30.466583251953324, 50.58112716674821]
+                        [30.466_304_779_052_763, 50.587_009_429_931_81], [30.466_583_251_953_324, 50.581_127_166_748_21]
```

Comment:
I would understand the rule for big decimals, but for floating point it looks weird. Nobody writes coordinates like this.

Reply:
:100: 
will try to configure the behaviour

Reply:
unfortunately, the rule is not customizable

Reply:
No, on the contrary, the rule is customizable -- PR added: https://github.com/kamu-data/kamu-cli/pull/491